QATCH/ui/mainWindow_ui.py

```
@@ -2528,3 +2654,203 @@ def emit(self, record):
+                        f"FAILURE - Port {next_port_num} NOT selected. Unexpected reply...")
+            else:
+                Log.e(
+                    f"TIMEOUT - Port {next_port_num} NOT selected. Controller timeout...")
+
+            if not success:
+                Log.d("Error Details: (serial response from port selection request)")
+                for line in flux_reply.splitlines():
+                    Log.d(f"ERROR >> {line}")
+                Log.d("Expected last line from controller to be \"Stepper: DONE!\"")
+
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)